File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -2485,28 +2204,25 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding a timestamp to existing PDF document
 
-You can add timestamp to the existing PDF document using the following code snippet.
+You can add timestamp to the existing PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property. The following code example explains the same.
```

Comment:
You can add timestamp to the existing PDF document using [TimeStampServer](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html#Syncfusion_Pdf_Security_PdfSignature_TimeStampServer) property in [PdfSignature]() class. The following code example explains the same.

Reply:
Changed the content